src/main/java/ui/MenuControl.java

```diff
@@ -353,6 +354,22 @@ private void updateRepoRemoveList(Menu remove) {
             removeItem.setOnAction(e1 -> onRepoRemove(repoId));
             remove.getItems().add(removeItem);
         }
+
+        remove.getItems().add(new SeparatorMenuItem());
+        MenuItem nonRemovableMsg = new MenuItem("[Non-Removable]");
```

Comment:
I was wondering if we should just put the `In Use` here so that we don't have to repeat it below?